lua/websocket/utilities.lua

```diff
@@ -38,7 +38,7 @@ end
 
 function utilities.HTTPHeaders(request)
 	assert(type(request) == "table", "parameter #1 is not a table")
-	assert(request[1] and find(request[1], ".*HTTP/1%.1"), "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
+	assert((request[1] and find(request[1], ".*HTTP/1%.1")) ~= nil, "parameter #1 (table) doesn't contain data or doesn't contain a HTTP request on key 1")
```

Comment:
I think it makes more sense to have it as `request[1] ~= nil and find(request[1], ".*HTTP/1%.1") ~= nil`.

Reply:
aye aye captain